packages/eas-cli/src/commands/open.ts

```diff
@@ -0,0 +1,45 @@
+import openBrowserAsync from 'better-opn';
+
+import { getProjectDashboardUrl } from '../build/utils/url';
+import EasCommand from '../commandUtils/EasCommand';
+import { ora } from '../ora';
+import { getExpoConfig } from '../project/expoConfig';
+import {
+  fetchProjectIdFromServerAsync,
+  findProjectRootAsync,
+  getProjectAccountName,
+} from '../project/projectUtils';
+import { ensureLoggedInAsync } from '../user/actions';
+
+export default class Open extends EasCommand {
+  static override description = 'open the project page in a web browser';
+
+  async runAsync(): Promise<void> {
+    const projectDir = await findProjectRootAsync();
+    const exp = getExpoConfig(projectDir);
+
+    // this ensures the project exists
+    await fetchProjectIdFromServerAsync(exp);
+
+    const user = await ensureLoggedInAsync();
+    const accountName = getProjectAccountName(exp, user);
+    const projectName = exp.slug;
+
+    const projectDashboardUrl = getProjectDashboardUrl(accountName, projectName);
```

Comment:
Following up on the internal discussion on projectId replacing account/slug as single unique project identifier, should we have an endpoint to retrieve this if `extra.eas.projectId` is specified, making sure that always takes precedence, instead of building more functionality on top of something we want to deprecate?

I know it's an existing util function, but I wonder if we should add another place where we use it.

Reply:
Good point, but I'd say this is only one additional line of code to replace. I don't think this particular PR is a good place to start the refactor.